pkg/policyengine/loadBalancer.go

```diff
@@ -177,7 +178,12 @@ func (lB *LoadBalancer) LookupECMP(service string, peers []string) (string, erro
 
 func (lB *LoadBalancer) LookupStatic(serviceSrc, serviceDst string, peers []string) (string, error) {
 	peer := lB.getDefaultPeer(serviceSrc, serviceDst)
-	plog.Infof("LookupStatic: serviceSrc %s serviceDst %s selects defaultPeer %s - target peer %s", serviceSrc, serviceDst, peer, peers)
+	plog.Infof(
```

Comment:
suggest moving message a line up if possible and minimizing line splits in parameters